src/core/vectortile/qgsmapboxglstyleconverter.cpp

```diff
@@ -1481,7 +1481,7 @@ void QgsMapBoxGlStyleConverter::parseSymbolLayer( const QVariantMap &jsonLayer,
             labelSettings.distUnits = context.targetUnit();
             labelSettings.dist = std::abs( textOffset.y() ) - textSize;
             labelSettings.lineSettings().setPlacementFlags( textOffset.y() > 0.0 ? QgsLabeling::BelowLine : QgsLabeling::AboveLine );
-            if ( textSizeProperty && textOffsetProperty.isEmpty() )
+            if ( !textSizeProperty )
```

Comment:
that's not right...

Reply:
```suggestion
            if ( textSizeProperty && !textOffsetProperty )
```

better :see_no_evil:

Reply:
```suggestion
            if ( textSizeProperty && !textOffsetProperty )
```

better :see_no_evil:

Reply:
yep